### ISSUE TITLE
battery:status doesn't work

### ISSUE THREAD

**Issue by saada** [guarinogabriel/Mac-CLI#65]
```bash
$ mac battery:status
system_profiler SPPowerDataType | awk '/Full/ || /Remaining/ || /Cycle/ { echo }' | sed -e 's/[^:]*/Battery/' -e 's/ *R.*):./\//' -e 's/ *Cycle/ (mAh)     Cycle/'


awk: illegal statement
 input record number 15, file
 source line number 1
```

OSX 10.11.6 Beta (15G24b)

**Comment by guarinogabriel** [comment]
@saada Could you please fix that command? I don't have the MacOS beta installed on my Macbook.

Please let me know.
Thanks.

**Comment by saada** [comment]
This seems to work for me:
```
system_profiler SPPowerDataType | awk '/Full/ || /Remaining/ || /Cycle/ {print}' | sed -e 's/[^:]*/Battery/' -e 's/ *R.*):./\//' -e 's/ *Cycle/ (mAh)     Cycle/'
Battery: 1526
Battery: No
Battery: 7422
Battery: 941
```
I change the {echo} to {print}.
Is that the expected output?

**Comment by saada** [comment]
If yes, then I'll go ahead and create a pull-request

**Comment by guarinogabriel** [comment]
Hi @saada,

That is almost correct. The only difference is that the commands was showing the right labels (instead of the "battery" four times).
Could you please get the labels fixed on the command? After that, you can create the PR and I will get it merged.

Thanks!
Gabriel

**Comment by saada** [comment]
With my very primitive awk skills:
```bash
$ system_profiler SPPowerDataType | awk '/Full/ || /Remaining/ || /Cycle/ {print}' | awk '{print $1 " " $2 " " $3 " " $4 " " $5}'
Charge Remaining (mAh): 7290
Fully Charged: Yes
Full Charge Capacity (mAh): 7381
Cycle Count: 941
```

**Comment by rgcr** [comment]
Hello everyone, I think  `pmset -g batt` can get the status easier  #69

**Comment by saada** [comment]
```bash
$ pmset -g batt
Now drawing from 'Battery Power'
 -InternalBattery-0 	47%; discharging; 1:58 remaining present: true
```

@rgcr , that seems like a different set of info. Maybe we can combine the two?

**Comment by guarinogabriel** [comment]
This has been fixed by @rgcr on PR #69
